src/renderer/components/App.tsx

```diff
@@ -64,6 +66,19 @@ export default class App extends React.Component<any, undefined> {
         ));
     }
 
+    public cleanTabFile(files: File[]): File[] {
```

Comment:
Coding advice: rather than "clean", use a more self-explanatory term such as "filterImportFilesByAllowedExtensions" (this is just an example, feel free to adapt :)